docs/management/api/get-endpoint-api.asciidoc

```diff
@@ -1,11 +1,141 @@
 [[get-endpoint-api]]
 === Get endpoint
 
-Enter API description here.
+Retrieves metadata about a single machine running {endpoint-sec}.
 
-NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
+NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Run Elasticsearch API requests].
```

Comment:
```suggestion
NOTE: Console supports Elasticsearch APIs only. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Run Elasticsearch API requests].
```

Reply:
Just wanted to clarify that this note is saying that Console will only support Elasticsearch APIs and no other APIs.